autopts/wid/bap.py

```diff
@@ -2015,11 +2027,12 @@ def hdl_wid_380(_: WIDParams):
                                              audio_locations, octets_per_frame,
                                              frames_per_sdu)
 
+    broadcast_id = 0x123456
     presentation_delay = 40000
     streams_per_subgroup = 2
     subgroups = 1
     broadcast_id = btp.bap_broadcast_source_setup(
-        streams_per_subgroup, subgroups, coding_format, vid, cid,
+        0, streams_per_subgroup, subgroups, coding_format, vid, cid,
```

Comment:
```suggestion
        streams_per_subgroup, subgroups, coding_format, vid, cid,
```

and then add `broadcast_id` as the last parameter